### PR TITLE
Change to multisample data

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-594

#### Link to staging deployment URL 

Staging link in UI https://github.com/biomage-ltd/ui/pull/132

#### Links to any Pull Requests related to this

https://github.com/biomage-ltd/ui/pull/132
https://github.com/biomage-ltd/inframock/pull/22

Relevant pull request : 
https://github.com/biomage-ltd/worker/pull/75 - move to Seurat object

In the PR, these are deleted :

- `test_count_matrix.py` deleted 
- `test_all_genes_returned_when_no_limit_is_specified` deleted

It causes problems in this MR because the result object as a test reference is not updated. Therefore, to pass the test, I deleted these files in the commit 2a8e1df.

#### Anything else the reviewers should know about the changes here

- New sample data have large file sizes `python.h5ad` (60.9 MB), `r.rds.gz` (198.5 MB), so they are stored in Git LFS.
- New experiment ID set to `e52b39624588791a7889e39c617f669e`

# Changes
### Code changes

- Created `EXPERIMENT_ID` as environment variables in `docker.yaml`
- Restored deleted `docker-compose.linux-dev.yaml` file to make it easeir for Linux users to pick up the project

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock
- [X] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [x] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by aerlaut** [comment]
The updated files show up as "empty files". However if I git clone to a new repository and pull this PR, the files are there. It's my first time using Git LFS. Is this an expected behaviour?

File: r/Dockerfile [hunk 1]
```diff
@@ -50,8 +50,5 @@ FROM builder AS dev
 # when source files change
 RUN pip install -U jedi radian PyYAML watchdog[watchmedo]
 
-ARG EXPERIMENT_ID
-ENV EXPERIMENT_ID $EXPERIMENT_ID
```
Comment (aerlaut) [inline r/Dockerfile:54]: Moved out as environment variables in `docker-compose.yml`

**Comment by marcellp** [comment]
It does not show up because the files are gzipped, so GitHub thinks it's a zip file that it does not want to print out. That said, the files do show up on my end as well in Git.

image

**Comment by marcellp** [comment]
Could we add some documentation about how to install LFS before working on the project?

**Comment by aerlaut** [comment]
> Could we add some documentation about how to install LFS before working on the project?

Sure, I'll write up about it in readme

**Review by marcellp — APPROVED** [review]
lgtm after documentation is updated

**Pull request merged**